test/unit/org/apache/cassandra/db/filter/ColumnFilterTest.java

```diff
@@ -86,7 +86,7 @@ public class ColumnFilterTest
     @Parameterized.Parameters(name = "{index}: clusterMinVersion={0}")
     public static Collection<Object[]> data()
     {
-        return Arrays.asList(new Object[]{ "3.0" }, new Object[]{ "3.11" }, new Object[]{ "4.0-rc1" }, new Object[]{ "4.0" });
+        return (Collection) Arrays.asList(new Object[]{ "4.0" });
```

Comment:
It seems that this test doesn't need to be parameterized anymore, since there is only a single parameter. The var `clusterMinVersion` can be removed.

Reply:
yeah you're right, my thinking applies really only to `*.net.*` tests

Reply:
done